src/annotations-ui/helpers/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './groups';
+export * from './mentions';
```

Comment:
I would prefer us to use explicit exports everywhere rather than namespace exports. This makes it easier to see what the exports are directly, and also avoids unintentionally exporting a symbol if it is exported only for internal usage.